### ISSUE TITLE
Stone Pressure Plate bookmark in NEI cause crash

### ISSUE THREAD

**Issue by Imagine0123** [GTNewHorizons/GT-New-Horizons-Modpack#16192]
### Your GTNH Discord Username

imagination12

### Your Pack Version

2.6.0

### Your Server

SP

### Java Version

Java 21

### Type of Server

Single Player

### Your Actions

Pinned the recipe for stone pressure plate and trying to connect it to the stone buttons recipe

### Crash Report

[crash-2024-05-06_21.18.59-client.txt](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/15221976/crash-2024-05-06_21.18.59-client.txt)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by slprime** [comment]
can't reproduce, need more details

**Comment by Imagine0123** [comment]
I'm sorry but when I restarted the game the crash also went away for some inexplicable reason. I bookmarked the OC keyboard recipe and the button recipe then linked them together. I went on to bookmark the pressure plate recipe and linked it to the previous recipe and then the client crashed. When I tried to enter the world again it immediately crashed the client again. I restarted the game and now it's all fine.
